python/ray/experimental/sgd/pytorch/utils.py

```diff
@@ -238,3 +238,8 @@ def sgd_mse_optimizer(model, config):
     criterion = nn.MSELoss()
     optimizer = torch.optim.SGD(model.parameters(), lr=learning_rate)
     return criterion, optimizer
+
+
+def clean_state_dict(state_dict):
+    """Removes the 'module.' prefix added by PyTorch"""
```

Comment:
this is only for distributed pytorch right?

Reply:
also this isn't called anywhere

Reply:
Thanks, forgot to remove this.